src/app/components/champion/ChampionView.tsx

```diff
@@ -2,14 +2,15 @@ import * as React from 'react';
 import {FC} from "react";
 import {
     Avatar,
-    Box,
+    Box, Fade,
```

Comment:
a linter+beautify would be nice to avoid this kind of formatting

Reply:
I've made a ticket for that (I also despise this formatting)